lts-8.yaml

```diff
@@ -10,10 +10,10 @@ packages:
 - zifter-stack
 
 extra-deps:
-- genvalidity-0.4.0.0
+- genvalidity-0.4.0.4
 - genvalidity-hspec-0.5.0.0
 - genvalidity-path-0.2.0.0
 - genvalidity-property-0.1.0.0
-- validity-0.4.0.0
+- validity-0.4.0.4
```

Comment:
This will probably get us into problems because validity wasn't available yet when this lts was made.